### DISCUSSION TITLE
share feedback about the project

### DISCUSSION THREAD

**Discussion by abinth11** [abinth11/TutorTrek#67] [Q&A]
share your valuable feedbacks

**Comment by nikhilnarayanan623** [comment]
I found it's following clean code architecture and I hope it helps new developers to build and learn projects

**Comment by Prakash21singh** [comment]
Bro, honestly this coding style is too good, i mean hats off to your knowledge, hoping for me to be a good developer like you one day.

This codebase is impressive.
